src/main/java/api/ApiClient.java

```diff
@@ -0,0 +1,24 @@
+package api;
+
+import io.restassured.response.ValidatableResponse;
+import org.apache.http.HttpHeaders;
+
+import static helpers.ConfigSetUp.getToken;
+
+public class ApiClient {
+
+    public ValidatableResponse execute(BaseRequest request) {
+        addDefaultHeaders(request);
+        authorize(request);
+        return request.execute();
+    }
+
+    private void addDefaultHeaders(BaseRequest request) {
+        request.headers.put("Content-Type", "application/json");
```

Comment:
Please, do not use direct access to the fields.

Reply:
implemented getters